app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1,6 +1,7 @@
 package org.fossasia.susi.ai.activities;
 
 import android.Manifest;
+import android.app.Activity;
```

Comment:
@anantprsd some errors with java.lang on the login screen there's a long log so i guess you should check it!